webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js

```diff
@@ -232,9 +232,10 @@ export class SakaiGrader extends gradableDataMixin(SakaiElement) {
                 ` : ""}
                 <div class="submitted-time ${this.submission.draft ? "draft-time" : ""}">${this.submission.submittedTime}</div>
                 ${this.submission.late ? html`<span class="grader-late">${this.assignmentsI18n["grades.lateness.late"]}</span>` : ""}
-                ${this.submission.returned ? html`<span class="grader-returned fa fa-eye" title="${this.i18n["returned_tooltip"]}" />` : ""}
+                ${this.submission.returned ? this.renderGraderReturned() : ""}
               ` : html`
                 <span>${this.i18n["no_submission_for"]} ${this.renderSubmitter()}</span>
+                ${this.renderGraderReturned()}
```

Comment:
I think that probably is a better aproach just move this line `${this.submission.returned ? html`<span class="grader-returned fa fa-eye" title="${this.i18n["returned_tooltip"]}" />` : ""`, because it doesn't have to do with the condition check that render this block.

I think you can just put this line up to the next `</div>` and outside the ternary operator that it is in currently, and then you can get rid of the new function.

Reply:
That woudn't work, this is the structure that would be created doing that

---

![image](https://user-images.githubusercontent.com/33053368/133746908-42cf0b31-38ec-4332-bff2-7e8f9c5a1912.png)

---

I'm putting the eye below the text like this

---

![image](https://user-images.githubusercontent.com/33053368/133747268-45fc7f10-6ede-48aa-b403-db313cb11184.png)

Reply:
Ok! Thanks! :D